src/modules/sub-items/sub.items.module.js

```diff
@@ -568,6 +585,34 @@ export default class SubItemsModule extends Component {
         this.bulkDelete();
     }
 
+    handleBulkOperationFailedNotification(selectedItems, failedLocations, modalTitle, notificationMessage) {
+        const { contentTypesMap } = this.state;
+        const failedItems = failedLocations.map(({ id: locationId }) => selectedItems.get(locationId));
```

Comment:
Do we need to `map` twice? Seems it can be done in one.

Reply:
Yes we can 😄